README.md

```diff
@@ -103,6 +107,37 @@ ssh:
 
 `task-environment`: The environment from which Orchestrator will serve task implementations. (default: `production`)
 
+### Node specific configruation with the inventory file.
+
+The inventory file allows you to group nodes and set up node specific configuration defaults. To set up default for ssh and winrm connections create and inventoryfile at `~/.puppetlabs/bolt/inventory.yaml` with the following content.
+
+```yaml
+groups:
```

Comment:
I'm not sure this actually works. We validate that every group has a name, and the top-level group here doesn't.

Reply:
Oh, I missed that we provide a default name of `all` for the top-level group. That'll be important on my PR.

Reply:
The top level group is given the name "all" https://github.com/puppetlabs/bolt/blob/master/lib/bolt/inventory.rb#L46